<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetAction</class>
 <widget class="QWidget" name="SetAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnUp">
         <property name="text">
          <string>Up</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/assets/images/button-icons/arrow-up.png</normaloff>:/assets/images/button-icons/arrow-up.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDown">
         <property name="text">
          <string>Down</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/assets/images/button-icons/arrow-down.png</normaloff>:/assets/images/button-icons/arrow-down.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="text">
          <string>Add</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/assets/images/button-icons/add.png</normaloff>:/assets/images/button-icons/add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemove">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/assets/images/button-icons/remove.png</normaloff>:/assets/images/button-icons/remove.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
